dist/index.js

```diff
@@ -7612,9 +7612,13 @@ async function get(url0, pageIdxs) {
       const headers = {
         'user-agent': 'setup-beam',
       }
-      if (process.env.GITHUB_TOKEN) {
-        headers.authorization = `Bearer ${process.env.GITHUB_TOKEN}`
+      const GithubToken =
```

Comment:
I was sure that the linter would complain about non-camelcase, but if it doesn't this is also Ok.